src/relayclient/RelayClient.ts

```diff
@@ -164,22 +195,10 @@ export class RelayClient {
     return false
   }
 
-  async init (): Promise<this> {
-    await this.initMutex.runExclusive(async () => {
-      if (this.initialized) {
-        return this
-      }
-      this.emit(new GsnInitEvent())
-      await this.contractInteractor.init()
-      this.initialized = true
-    })
-    return this
-  }
-
   async relayTransaction (gsnTransactionDetails: GsnTransactionDetails): Promise<RelayingResult> {
     if (!this.initialized) {
       if (!this.initMutex.isLocked()) {
-        this._warn('better call RelayClient.init() in advance (to make first request faster)')
+        this._warn('suggestion: call RelayProvider.init()/RelayClient.init() in advance (to make first request faster)')
```

Comment:
I think we should remove this flow. Users should just do `new RelayClient().init().then(enableSendButton)` if needed - the elapsed time will be the same.
What do you think?

Reply:
I don't want to enforce it.
I want the provider to behave ok even if you don't call init()
There are scenarios where is not that easy to enable/disable the ui, or use async call.